ee/fleet-agent-downloader/api/controllers/entrance/signup-sso-user-or-redirect.js

```
@@ -0,0 +1,51 @@
+    if(!sails.config.custom.ssoClientSecret) {
+      throw {redirect: '/login'};
+    }
+    if(!this.req.session) {// If the requesting user does not have a session, redirect them to the login page.
+      throw {redirect: '/login'};
+    }
+    // If the requesting user has a session, but it does not contain a passport object, we'll redirect them to the login page.
+    if (!this.req.session.passport || !this.req.session.passport.user) {
+      throw {redirect: '/login'};
+    }
+
+    let ssoUserInfo = this.req.session.passport.user.userinfo;
```

ESLint

[error] 'ssoUserInfo' is never reassigned. Use 'const' instead.

(prefer-const)